libqtile/bar.py

```diff
@@ -308,6 +308,10 @@ def _configure(self, qtile, screen, reconfigure=False):
             )
             self.qtile.renamed_widgets.clear()
 
+        if self.qtile.core.name == "x11":
+            hook.subscribe.setgroup(self.keep_below)
+            hook.subscribe.startup_complete(self.keep_below)
```

Comment:
TBH this is general user logic that I guess isn't X-specific? Probs fine to drop the backend check:
```suggestion
        hook.subscribe.setgroup(self.keep_below)
        hook.subscribe.startup_complete(self.keep_below)
```